smarts/core/sensors.py

```diff
@@ -925,14 +925,18 @@ def teardown(self):
 
 
 class WaypointsSensor(Sensor):
-    def __init__(self, vehicle, mission_planner, lookahead=50):
+    def __init__(self, vehicle, mission_planner, sim, lookahead=50):
```

Comment:
Can we put sim at the beginning of the signature? I believe that's how most of the code has been doing it. (just for style)